examples/playbooks/unicode.transformed.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
[This](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/unicode.transformed.yml) is a reformatted copy of [examples/playbooks/unicode.yml](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/unicode.yml)